### PR TITLE
[1534] Added support for .woff and more

### PR DESCRIPTION
**Pull request by stanica**

### CODE REVIEW

File: css/webfonts.less [hunk 1]
```diff
@@ -3,27 +3,45 @@
 */
 
 @font-face {
-  font-family: 'Open Sans';
-  font-style: italic;
-  font-weight: 400;
-  src: local('Open Sans Italic'), local('Open-Sans-Italic'),
-    url('../resources/fonts/OpenSans-Italic.ttf') format('truetype');
+    font-family: 'Open Sans';
+    src: url('opensans-bold-webfont.eot');
```
Comment (jbuck) [inline css/webfonts.less]: for this src attribute and all others, only include the .woff and .ttf versions of files.

**Comment by jbuck** [comment]
In the rebase, make sure you delete your merge commits as well

File: css/webfonts.less [hunk 2]
```diff
@@ -3,27 +3,33 @@
...
 @font-face {
-  font-family: 'Open Sans';
-  font-style: normal;
-  font-weight: 400;
-  src: local('Open Sans Regular'), local('Open-Sans-Regular'),
-    url('../resources/fonts/OpenSans-Regular.ttf') format('truetype');
+    font-family: 'Open Sans';
+    src: url('opensans-regular-webfont.woff') format('woff'),
+         url('opensans-regular-webfont.ttf') format('truetype');
+}
```
Comment (jbuck) [inline css/webfonts.less:25]: Do you need to specify `font-style: normal` here or is that the default?

Comment (k88hudson) [inline css/webfonts.less:25]: That is the default; not necessary.

File: css/webfonts.less [hunk 3]
```diff
@@ -3,27 +3,33 @@
...
 @font-face {
-  font-family: 'Open Sans';
-  font-style: normal;
-  font-weight: 700;
-  src: local('Open Sans Bold'), local('Open-Sans-Bold'),
-    url('../resources/fonts/OpenSans-Bold.ttf') format('truetype');
+    font-family: 'Open Sans';
+    src: url('opensans-bolditalic-webfont.woff') format('woff'),
+         url('opensans-bolditalic-webfont.ttf') format('truetype');
+    font-weight: 700;
```
Comment (jbuck) [inline css/webfonts.less:17]: what about `font-weight: bold` instead? Kate, is there any functional difference?

File: css/webfonts.less (hunk 1 above)
Comment (k88hudson) [inline css/webfonts.less]: +1

File: css/webfonts.less (hunk 3 above)
Comment (k88hudson) [inline css/webfonts.less:17]: No difference.

**Comment by mjschranz** [comment]
Staged on master: https://github.com/mozilla/butter/commit/7150a57859c67edd97f213a461910c91db8303ff

**Pull request closed without merging**